src/style.css

```diff
@@ -0,0 +1,154 @@
+:root {
+  --main_text: #060b35;
+  --secondary_text: #616070;
+  --design_color: #00acdc;
+  --background_color_buy: #fff;
+  --left_position: calc(50% - 166px / 2);
+  --width: 166px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background-color: var(--background_color_buy);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img-imac {
+  background-image: url(./images/imac.jpeg);
+  background-size: 160px 134px;
+  width: 160px;
+  height: 134px;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: absolute;
```

Comment:
it's not a good practice to make all of you elements positioned using position:absolute. 
Just add paddings for your card and make indents between element by using margin

Reply:
done